functional-samples/cookbook.sidepanel-action/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "manifest_version": 3,
+  "name": "VIOLET google.com side panel example",
```

Comment:
Thoughts on removing the colours from the name? I love the icons but I think having it in the name looks a bit strange if you're not used to it.

Reply:
DOne! Yep that was mostly for my own sanity as I was testing these 😸